package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-ui-react",
-  "version": "1.1.0",
```

Comment:
I'm not quite sure what happened but isn't this 1.3.0-beta.1 on the latest develop branch? https://github.com/yext/search-ui-react/blob/develop/package.json#L3